modules/cnf-topology-aware-lifecycle-manager-precache-user-spec-images.adoc

```diff
@@ -0,0 +1,259 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc
+
+:_content-type: PROCEDURE
+[id="talm-prechache-user-specified-images_{context}"]
+= Pre-caching user specified images with {cgu-operator} on {sno} clusters
+
+In earlier {product-title} versions, you could pre-cache {product-title} workloads on {sno} clusters in a cluster with limited bandwidth.
+You can also pre-cache application-specific workload images on {sno} clusters before upgrading your applications.
+
+With the `PreCachingConfig` custom resource (CR) referenced in the `ClusterGroupUpgrade` CR, you can specify the configuration options for the pre-caching jobs.
```

Comment:
So when I first read this I wrote the following:

> I'd be inclined to think that this is a two-step process
> 1)  Specify the configuration options for the pre-caching jobs in the`PreCachingConfig` CR
> 2)  Update the `ClusterGroupUpgrade` CR to include the reference to the `PreCachingConfig`
> 
> So you might think about laying it out that way, reversing the order in which you present the example yaml files and introducing each with an instruction along the lines of : create/edit the CR as follows.
> 

Then  I scrolled down and found that you had done exactly that.
But I'm wondering why you gave this kind of preview of the two CRs.. The first examples are more detailed than the ones used in the procedure steps so maybe that's the reason.
If you want to keep this structure then I'd suggest rewriting the intro sentence. 
Keeping it simple would be:
You can specify the configuration options for the pre-caching jobs using the following custom resources (CR): 
- `PreCachingConfig`
- `ClusterGroupUpgrade`

Then present the examples as they are but I'd suggest the `PreCachingConfig` should come first